option.go

```diff
@@ -241,26 +247,46 @@ func WithJwkErrRetryInterval(i string) Option {
 	access token parameters
 */
 
-// WithATEnableMTLSCertificateBoundAccessToken returns a EnableMTLSCertificateBoundAccessToken functional option
-func WithATEnableMTLSCertificateBoundAccessToken(b bool) Option {
+// WithNewATProcessor returns a functional option that appends the new access token processor parameters
```

Comment:
WithNewATProcessor → WithNewATProcessorParam ?

Reply:
continue in https://github.com/yahoojapan/athenz-authorizer/pull/59#discussion_r402141793